### PR TITLE
fix: edit presubmit for to simplify configuration

### PR DESCRIPTION
**Pull request by chalmerlowe**
This creates a new cfg that could be used in place of the existing kokoro presubmit test.
The intent here is to avoid using variables to skip certain nox sessions and simply call the nox sessions that we want to execute. The benefit is that...because nox sessions create the virtual env before they execute the session steps, we end up needlessly creating virtual envs for sessions that we end up skipping when we reach the skip/not skip logic.

### CODE REVIEW

File: .kokoro/presubmit/presubmit-2.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Only run these nox sessions.
+env_vars: {
+    key: "NOX_SESSION"
+    value: "unit_noextras-3.7 unit_noextras-3.12 unit-3.7 unit-3.8 unit-3.12 cover docs"
```
Comment (chalmerlowe) [inline .kokoro/presubmit/presubmit-2.cfg:6]:
```suggestion
    value: "unit_noextras-3.7" # unit_noextras-3.12 unit-3.7 unit-3.8 unit-3.12 cover docs"
```

File: .kokoro/presubmit/presubmit-2.cfg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Only run these nox sessions.
+env_vars: {
+    key: "NOX_SESSION"
+    value: "unit_noextras-3.7" # unit_noextras-3.12 unit-3.7 unit-3.8 unit-3.12 cover docs"
```
Comment (chalmerlowe) [inline .kokoro/presubmit/presubmit-2.cfg:6]:
```suggestion
    value: "unit_noextras-3.7 unit_noextras-3.12" # unit-3.7 unit-3.8 unit-3.12 cover docs"
```

File: .kokoro/presubmit/presubmit-2.cfg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Only run these nox sessions.
+env_vars: {
+    key: "NOX_SESSION"
+    value: "unit_noextras-3.7 unit_noextras-3.12" # unit-3.7 unit-3.8 unit-3.12 cover docs"
```
Comment (chalmerlowe) [inline .kokoro/presubmit/presubmit-2.cfg:6]:
```suggestion
    value: "unit_noextras-3.7 unit_noextras-3.12 unit-3.7 unit-3.8 unit-3.12 cover docs"
```

**Comment by chalmerlowe** [comment]
Using base names in the config file works fine.
`unit` in the config will lead to `unit-3.7` and `unit-3.12` (whatever versions of Python are fed into the `nox.session`) in the results.

File: noxfile.py [hunk 4]
```diff
@@ -56,6 +56,9 @@
     "docs",
 ]
 
+import time
+from functools import wraps
+
```
Comment (chalmerlowe) [inline noxfile.py:61]:
```suggestion
```

**Review by shollyman — APPROVED** [review]
the title here seems out of sync with the changes, but I may be missing some context. It appears to simply be an update to the existing config, not a new one.

**Review by Linchin — APPROVED** [review]

**Pull request merged**